Portfolio mobile.html

```diff
@@ -7,18 +7,196 @@
+                <li><img src="./images/Rectangle%2070.png" alt="split"/></li>
+                <li>Ruby</li>
+            </ul>
+        <button class="button" type="button">See Project</button>
+    </div>
+    </article>
+    </section>
+
+    <section class="Project_work4">
+        <article class="article">
+        <div class="workform">
+        <h6 class="posted">Multi-Post Stories</h6>
+        <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required. 
+            has been the industry's standard dummy text ever since the 1500s, 
+            when an unknown printer took a standard dummy text.</p>
+            <ul class="techstack">
+                <li>css</li>
+                <li><img src="./images/Rectangle%2070.png" alt="split"/></li>
+                <li>html</li>
+                <li><img src="./images/Rectangle%2070.png" alt="split"/></li>
+                <li>bootstrap</li>
+                <li><img src="./images/Rectangle%2070.png" alt="split"/></li>
+                <li>Ruby</li>
+            </ul>
+        <button class="button" type="button">See Project</button>
+    </div>
+    <img class="Placeholder" src="./images/Img%20Placeholder.png" alt="Image place holder"/>
+    </article>
+    </section>
+    <footer class="Contact-form">
+        <h2>Let's chat!</h2>
+        <p>If you have an application you are interested in developing, 
+                a feature that you need built or a project that needs coding. 
+                I'd love to help with it 
+        </p>
+        <form class="form-info" action="https://formspree.io/f/mzbooyno" method="post">
+            <div class="Name">
+                <input type="text" name="fname" placeholder="Enter first name here" required maxlength="30"/>
+                <input type="text" name="lname" placeholder="Enter last name here" required maxlength="30"/>
+            </div>
+                <input type="email" name="email" placeholder="Enter your email address" required pattern="^([a-zA-Z0-9_\-\.]+)@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.)|(([a-zA-Z0-9\-]+\.)+))([a-zA-Z]{2,4}|[0-9]{1,3})(\]?)$"/>
+                <textarea name="messagebox" placeholder="Enter text here" rows="10" cols="30" required maxlength="500"></textarea>
+            <button class="button" type="submit">Send</button>
+        </form>
+        <img id="large-divider" src="./images/Rectangle%2042.png" alt="final divider"/>
+        <div class="icons">
+            <a><img src="./images/social1.png" alt="social1"/></a>
+            <a><img src="./images/social2.png" alt="social2"/></a>
+            <a><img src="./images/social3.png" alt="social3"/></a>
+            <a><img src="./images/social4.png" alt="social4"/></a>
+            <a><img src="./images/social5.png" alt="social5"/></a>
+            <a id="email" href="#">shuhad.loofer@gmail.com</a>
+        </div>
+</footer>
+</div>
+
+                                    <!-- end of desktop version html -->
```

Comment:
- [ ] For best practices, please consider using CSS to style the page instead of re-creating the HTML code for mobile